crl/crl.go

```diff
@@ -50,6 +50,7 @@ var (
 	oidISOSignatureSHA1WithRSA = asn1.ObjectIdentifier{1, 3, 14, 3, 2, 29}
 )
 
+// RFC 5280, 5.1.1.2
```

Comment:
You can link to a specific section number like so: https://tools.ietf.org/html/rfc5280#section-5.1.1.2.

Also, I'd like to see more context on what this reference means. For instance, one might be led to believe that this struct directly corresponds to the `signatureAlgorithm` field defined in section 5.1.1.2, which is not the case.

I'd say something like "This list allows us to build an RFC 5280-style AlgorithmIdentifier from one of Go's internal algorithm constants."